### PR TITLE
Different searches

### PR DESCRIPTION
**Pull request by Dcox270891**
Cleaned up the code
moved the map into the script.js and made sure it is all working
added to search by ratings cost and distance and remove the old results
removed the get location button as the googlemaps api is already getting the location

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -61,9 +64,10 @@ modalClose.addEventListener("click", function(event) {
...
 });
 
-searchButton.addEventListener(`click`, event => {
+searchDistance.addEventListener(`click`, event => {
+    clearResults ()
     fetch(
-            `https://developers.zomato.com/api/v2.1/search?lat=${userLatitude}&lon=${userLongitude}&apikey=969dccb114a560b6d4df35b25a8e6418`
+            `https://developers.zomato.com/api/v2.1/search?lat=${userLatitude}&lon=${userLongitude}&apikey=969dccb114a560b6d4df35b25a8e6418&sort=real-distance`
         )
         .then(repsonse => repsonse.json())
```
Comment (NalinKane) [inline script.js]: there is a typo here - should be "response" and not "repsonse"

File: script.js [hunk 2]
```diff
@@ -82,26 +86,112 @@ searchButton.addEventListener(`click`, event => {
...
+            });
+        });
+});
 
+searchPrice.addEventListener(`click`, event => {
+    clearResults ()
+    fetch(
+            `https://developers.zomato.com/api/v2.1/search?lat=${userLatitude}&lon=${userLongitude}&apikey=969dccb114a560b6d4df35b25a8e6418&sort=cost`
+        )
+        .then(repsonse => repsonse.json())
```
Comment (NalinKane) [inline script.js]: same here - but somehow the code is still working??

File: script.js [hunk 3]
```diff
@@ -82,26 +86,112 @@ searchButton.addEventListener(`click`, event => {
...
             });
         });
+});
+
+searchRating.addEventListener(`click`, event => {
+    clearResults ()
+    fetch(
+            `https://developers.zomato.com/api/v2.1/search?lat=${userLatitude}&lon=${userLongitude}&apikey=969dccb114a560b6d4df35b25a8e6418&sort=rating`
+        )
+        .then(repsonse => repsonse.json())
```
Comment (NalinKane) [inline script.js]: and here as well

**Review by NalinKane — APPROVED** [review]

File: script.js (hunk 3 above)
Comment (Dcox270891) [reply script.js]: Ive made the changes to the spelling mistakes

File: script.js (hunk 2 above)
Comment (Dcox270891) [reply script.js]: Ive made the changes to the spelling mistakes

File: script.js (hunk 1 above)
Comment (Dcox270891) [reply script.js]: Ive made the changes to the spelling mistakes

**Review by jennyunchan — COMMENTED** [review]
i've noticed that the button element for the get-location id has been removed from the html file but the queryselector code for get-location has been kept in the script.js file. is the button still required?

<button class="button" id="get-location">Get your location</button>

const getLocationButton = document.querySelector(`#get-location`);

**Comment by Dcox270891** [comment]
I took it out because its getting the location for google maps instantly as soon as the page is opened i will take out the query selector for it too

**Comment by Dcox270891** [comment]
Removed the query selector

**Review by jennyunchan — APPROVED** [review]

**Pull request merged**